portal/src/components/roles-and-groups/list/common/RolesAndGroupsBaseList.tsx

```diff
@@ -50,6 +50,15 @@ function RolesAndGroupsBaseList<T>(
     emptyText,
   } = props;
 
+  // NOTE: Avoid DetailsList automatically take key column of item as key of react list, causing duplicated key error
+  // Ref https://github.com/microsoft/fluentui/blob/19195df9f17f287bf4c66dd30453dd0d0a0ced93/packages/react/src/components/DetailsList/DetailsList.base.tsx#L1499
+  const getKey = useCallback((item: any, index?: number) => {
+    const baseKey = index?.toString() ?? Math.random().toString();
```

Comment:
We should not generate random string as key, because that defeat the purpose of "key".
What about simply fallback to 0? (Because index should always exist?)